io.neovim.nvim.tool.podman.yml

```diff
@@ -0,0 +1,45 @@
+id: io.neovim.nvim.tool.podman
+branch: "22.08"
```

Comment:
are you sure?

Reply:
Should I remove `branch: "22.08"` since I removed it in Neovim?

Reply:
yeah. this comment came before the one on the neovim PR.

Reply:
Done!